getyour/supervisord.conf

```diff
@@ -4,9 +4,16 @@ nodaemon=true
 [program:django]
 command=python manage.py runserver 0.0.0.0:8000
 autorestart=true
-priority=10
+priority=300
```

Comment:
It definitely makes sense to me to have `redis` start up first, but should `django` have a lower priority / start before `django-q` since it governs it?

Reply:
This is a great question! I knew this technically worked, but I didn't actually know why until trying to answer your question. There are two parts here: 

1) How does this configuration technically work? 
2) Why did I set it up this way to begin with?

### How does this configuration technically work?

When we run the command `python manage.py qcluster`,  the `DJANGO_SETTINGS_MODULE` environment variable is set to point to the Get-Your project's settings module. This loads our settings (like the `Q_CLUSTER` configuration) into the environment. The `qcluster` management command retrieves the `Q_CLUSTER` configuration from our Django settings, configures the cluster, connects to the broker, and is ready to start processing tasks. All without having to start the Django app.

### Why did I set it up this way to begin with?

One point you brought up when we talked about caching was that redeployments of the app will wipeout the cache. What I've done with this PR, is created a custom event to emit when the Django app starts. This event will trigger a task to run that repopulates the cache. My thought was to have the cluster running, so that when the Django app started it would be ready to process the task and repopulate the cache.

Reply:
Oh, very cool! Excellent addition, and thank you for the explanation.